fix(docs): meta tags + edit link

olarclara opened the pull request:
### Replaces references to `Nextra` in the meta tags.

Right now, when we share links to the docs, the generated previews mention Nextra instead of CodeSandbox:

![Screen Shot 2022-12-01 at 15 10 37](https://user-images.githubusercontent.com/24959348/205128572-430cf24d-203f-46bc-a109-ac95b77c2988.png)

After this gets merged, it will show the same content as when we share the website:

**Title**: CodeSandbox: Online Code Editor and IDE for Rapid Web Development
**Description**: CodeSandbox is an online code editor and prototyping tool that makes creating and sharing web apps faster

cc @filipelima18 if we want to change the content.

The Twitter user was also pointing to the library's creator instead of @codesandbox.

## Fixes the "Edit this page on CodeSandbox" link:

It currently builds a wrong URL leading to a broken experience when users try to edit the documentation by opening it in CodeSandbox.

**Current**:

https://user-images.githubusercontent.com/24959348/205129468-d70f82bf-414d-4e23-b662-9f39303bc4a2.mov

**Fixed**:


https://user-images.githubusercontent.com/24959348/205130152-20ade1c9-9131-499e-9974-9e268b442642.mov

zehfernandes commented on packages/codesandbox-theme-docs/dist/index.js:
```diff
@@ -495,7 +495,12 @@ var getGitEditUrl = (filePath) => {
   if (!repo)
     throw new Error("Invalid `docsRepositoryBase` URL!");
-  return `${repo.href}/${filePath}`;
+  console.log(repo.href);
```
console leftover?

olarclara replied:
Nice catch. Removed here: [ab01e68](https://github.com/codesandbox/docs/pull/101/commits/ab01e68ed3c76099f95cbbb2721b41bb82671e4d). I think we should take some time to add formatting and linting to this project.

olarclara commented:
@zehfernandes could you elaborate? I looked at the dist output and seems that the values defined on the constants are taking precedence over the values declared on the shared config file. Also, the constants are (at least should be) type-safe, so it looks like a better option to make sure that the properties are expected.

zehfernandes commented:
The `codesandbox-theme-docs` is a fork from `nextra-theme-docs`, according to their documentation we should be able to override all constants with a `theme.config.js` file.

We are doing this here https://github.com/codesandbox/docs/blob/main/packages/projects-docs/theme.config.js

I'm wondering why the `dist output` is not getting the correct values, probably a bug in the `nextra-theme-docs`?

olarclara commented:
@zehfernandes just discovered the prop in `theme.config.js` was not correct, changed from `getNextSeoProps ` to `head`, and removed the `head` value from the `constants` file. It works as expected now: https://projects-docs-4h4r78fj1-codesandbox1.vercel.app/docs/learn/introduction/overview.

codebryo commented:
Awesome work!
